tests/integration/it_utils.py

```
@@ -636,3 +641,207 @@ def create_mpt_token_and_authorize_source(
+    mp_token_issuance = MPTokenIssuanceCreate(
+        account=issuer.classic_address,
+        flags=flags,
+    )
+
+    tx_resp = await sign_and_reliable_submission_async(
+        mp_token_issuance, issuer, client=client
+    )
+    if tx_resp.result["engine_result"] != "tesSUCCESS":
+        raise XRPLException(
+            f"Unable to execute MPTokenIssuanceCreate Transaction: {tx_resp}"
+        )
```

Ruff (0.15.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)